### PR TITLE
DOCS Add audit ignore policy settings to auditing reference

### PR DESCRIPTION
**Pull request by albertzaharovits**
Adds the logfile audit ignore policy settings synopsis to the Auditing Reference page.

### CODE REVIEW

File: docs/reference/settings/audit-settings.asciidoc [hunk 1]
```diff
@@ -87,6 +84,36 @@ changes the setting in the config file, the node id will persist across cluster
 restarts and the administrator cannot change it.
 The default value is `true`.
 
+[[audit-event-ignore-policies]]
+==== Audit Logfile Event Ignore Policies
+
+These settings enable fine grained control over which audit events
```
Comment (lcawl) [inline docs/reference/settings/audit-settings.asciidoc]:
```suggestion
These settings affect the {stack-ov}/audit-log-output.html#audit-log-ignore-policy[ignore policies] that enable fine grained control over which audit events
```

File: docs/reference/settings/audit-settings.asciidoc [hunk 2]
```diff
@@ -87,6 +84,36 @@ changes the setting in the config file, the node id will persist across cluster
...
+are eventually printed to the log file. See {xpack-ref}/audit-log-output.html#audit-log-ignore-policy[Logfile Audit Events Ignore Policies]
```
Comment (lcawl) [inline docs/reference/settings/audit-settings.asciidoc]:
```suggestion
are printed to the log file. 
```

File: docs/reference/settings/audit-settings.asciidoc [hunk 3]
```diff
@@ -87,6 +84,36 @@ changes the setting in the config file, the node id will persist across cluster
...
+for the gory details.
```
Comment (lcawl) [inline docs/reference/settings/audit-settings.asciidoc]:
```suggestion

```

File: docs/reference/settings/audit-settings.asciidoc [hunk 4]
```diff
@@ -87,6 +84,36 @@ changes the setting in the config file, the node id will persist across cluster
...
+
+`xpack.security.audit.logfile.events.ignore_filters.<policy_name>.users`::
+Binds with other settings with the same `<policy_name>` to form a policy.
```
Comment (lcawl) [inline docs/reference/settings/audit-settings.asciidoc]:
```suggestion

```

File: docs/reference/settings/audit-settings.asciidoc [hunk 5]
```diff
@@ -87,6 +84,36 @@ changes the setting in the config file, the node id will persist across cluster
...
+The value is a list of names or wildcards, and specifies that the policy will
+not print audit events for users matching any of the specifed names or wildcards.
+`xpack.security.audit.logfile.events.ignore_filters.<policy_name>.realms`::
+Binds with other settings with the same `<policy_name>` to form a policy.
```
Comment (lcawl) [inline docs/reference/settings/audit-settings.asciidoc]:
```suggestion
 
```

File: docs/reference/settings/audit-settings.asciidoc [hunk 6]
```diff
@@ -87,6 +84,36 @@ changes the setting in the config file, the node id will persist across cluster
...
+The value is a list of names or wildcards, and specifies that the policy will
+not print audit events for users whose authentication realm matches any of the
+specified names or wildcards.
+`xpack.security.audit.logfile.events.ignore_filters.<policy_name>.roles`::
+Binds with other settings with the same `<policy_name>` to form a policy.
```
Comment (lcawl) [inline docs/reference/settings/audit-settings.asciidoc]:
```suggestion

```

File: docs/reference/settings/audit-settings.asciidoc [hunk 7]
```diff
@@ -87,6 +84,36 @@ changes the setting in the config file, the node id will persist across cluster
...
+The value is a list of names or wildcards, and specifies that the policy will
+not print audit events for users whose roles match any of the specified
+names or wildcards. If the user has several roles, some of which are *not*
+covered by the policy, then the policy will *not* cover this event.
+`xpack.security.audit.logfile.events.ignore_filters.<policy_name>.indices`::
+Binds with other settings with the same `<policy_name>` to form a policy.
```
Comment (lcawl) [inline docs/reference/settings/audit-settings.asciidoc]:
```suggestion

```

File: docs/reference/settings/audit-settings.asciidoc [hunk 8]
```diff
@@ -87,6 +84,36 @@ changes the setting in the config file, the node id will persist across cluster
...
+[[audit-event-ignore-policies]]
+==== Audit Logfile Event Ignore Policies
+
+These settings enable fine grained control over which audit events
+are eventually printed to the log file. See {xpack-ref}/audit-log-output.html#audit-log-ignore-policy[Logfile Audit Events Ignore Policies]
+for the gory details.
+
+`xpack.security.audit.logfile.events.ignore_filters.<policy_name>.users`::
+Binds with other settings with the same `<policy_name>` to form a policy.
+The value is a list of names or wildcards, and specifies that the policy will
```
Comment (lcawl) [inline docs/reference/settings/audit-settings.asciidoc]:
```suggestion
A list of user names or wildcards. The specified policy will
```

File: docs/reference/settings/audit-settings.asciidoc [hunk 9]
```diff
@@ -87,6 +84,36 @@ changes the setting in the config file, the node id will persist across cluster
...
+not print audit events for users matching any of the specifed names or wildcards.
```
Comment (lcawl) [inline docs/reference/settings/audit-settings.asciidoc]:
```suggestion
not print audit events for users matching these values.
```

File: docs/reference/settings/audit-settings.asciidoc [hunk 10]
```diff
@@ -87,6 +84,36 @@ changes the setting in the config file, the node id will persist across cluster
...
+`xpack.security.audit.logfile.events.ignore_filters.<policy_name>.realms`::
+Binds with other settings with the same `<policy_name>` to form a policy.
+The value is a list of names or wildcards, and specifies that the policy will
```
Comment (lcawl) [inline docs/reference/settings/audit-settings.asciidoc]:
```suggestion
A list of authentication realm names or wildcards. The specified policy will
```

File: docs/reference/settings/audit-settings.asciidoc [hunk 11]
```diff
@@ -87,6 +84,36 @@ changes the setting in the config file, the node id will persist across cluster
...
+not print audit events for users whose authentication realm matches any of the
```
Comment (lcawl) [inline docs/reference/settings/audit-settings.asciidoc]:
```suggestion
not print audit events for users in these realms.
```

File: docs/reference/settings/audit-settings.asciidoc [hunk 12]
```diff
@@ -87,6 +84,36 @@ changes the setting in the config file, the node id will persist across cluster
...
+specified names or wildcards.
```
Comment (lcawl) [inline docs/reference/settings/audit-settings.asciidoc]:
```suggestion

```

File: docs/reference/settings/audit-settings.asciidoc [hunk 13]
```diff
@@ -87,6 +84,36 @@ changes the setting in the config file, the node id will persist across cluster
...
+`xpack.security.audit.logfile.events.ignore_filters.<policy_name>.roles`::
+Binds with other settings with the same `<policy_name>` to form a policy.
+The value is a list of names or wildcards, and specifies that the policy will
```
Comment (lcawl) [inline docs/reference/settings/audit-settings.asciidoc]:
```suggestion
A list of role names or wildcards. The specified policy will
```

File: docs/reference/settings/audit-settings.asciidoc (hunk 3 above)
Comment (albertzaharovits) [reply docs/reference/settings/audit-settings.asciidoc]: good call.

**Comment by albertzaharovits** [comment]
Thank you @lcawl . I did some reformatting and checked the build. Looks fine!

**Review by lcawl — APPROVED** [review]
I added one more commit then verified the output. LGTM

**Comment by albertzaharovits** [comment]
run gradle build tests

**Comment by albertzaharovits** [comment]
run gradlee build tests

**Comment by albertzaharovits** [comment]
run gradle build tests

**Comment by albertzaharovits** [comment]
run gradle build tests

**Comment by albertzaharovits** [comment]
tough day for CI,
run gradle build tests

**Comment by albertzaharovits** [comment]
test this please

**Pull request merged**